src/c++/perf_analyzer/schema.json

```diff
@@ -0,0 +1,96 @@
+    "type": "object",
+    "properties": {
+        "experiments": {
+            "description": "The array of all experiments run by perf analyzer",
+            "type": "array",
+            "items": {
+                "type": "object"
+            },
+            "minItems": 1,
+            "uniqueItems": true,
+            "properties": {
+                "experiment": {
+                    "description": "A single experiment run by perf analyzer",
+                    "type": "object",
+                    "minItems": 1,
+                    "maxItems": 1,
+                    "properties": {
+                        "mode": {
+                            "description": "Operating mode of perf analyzer: either concurrency or request rate",
+                            "type": "string"
+                        },
+                        "value": {
+                            "description": "Concurrency or request rate for the current experiment",
+                            "type": "number"
+                        }
+                    },
+                    "required": [
+                        "mode",
+                        "value"
+                    ]
+                },
+                "requests": {
+                    "description": "The array of requests sent by perf_analyzer for this experiment",
+                    "type": "array",
+                    "uniqueItems": true,
+                    "items": {
+                        "$ref": "#/properties/experiments/properties/$defs/request"
+                    }
+                },
+                "$defs": {
+                    "request": {
+                        "description": "Info for a single request",
+                        "type": "object",
+                        "properties": {
+                            "timestamp": {
+                                "description": "Time stamp of the request",
+                                "type": "number"
+                            },
+                            "sequence_id": {
+                                "description": "sequence_id of the request",
+                                "type": "number"
+                            },
+                            "responses_timestamps": {
+                                "description": "All associated responses to this request",
+                                "type": "array",
+                                "items": {
+                                    "type": "number"
+                                }
+                            }
+                        },
+                        "required": [
+                            "timestamp",
+                            "responses_timestamps"
+                        ]
+                    }
+                },
+                "window_boundaries": {
+                    "description": "An array of time stamps describing window boundaries",
```

Comment:
Do we have a word for the type of window?  Is it an experiment window?  Measurement window?